js/dropdown.js

```diff
@@ -34,6 +34,13 @@
     return $parent && $parent.length ? $parent : $this.parent()
   }
 
+  // INPUT TYPE CHECK
+  // ================
+  function IsEditableElement(e) {
+    return /input|textarea/i.test(e.target.tagName)
+      || e.target.attributes.getNamedItem('contenteditable')
```

Comment:
Actually, what about https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/isContentEditable ? That seems to handle descendants of contentEditable elements properly.